waku/waku.md

```diff
@@ -510,6 +523,15 @@ Known static nodes MAY also be used.
 
 ## Changelog
 
+### Version 0.4
+
+Released [February 20, 2020](https://github.com/vacp2p/specs/commit/568ae72e3a438fecd21e85fc34b05fd5c3bb4bcf).
+
+- Add Status Update code `0x02`
+- Introduces a new packet code Status Code (`0x02`) for communicating option changes
+- Breaking: Removes support for packet codes `0x02`, `0x03`, `0x20`; superseded by `0x02`
```

Comment:
```suggestion
- Breaking: Removes support for the following packet codes: PoW Requirement (`0x02`), Bloom Filter (`0x03`), Rate limits (`0x20`), Topic interest (`0x21`) - all superseded by the new Status Code (`0x02`)
```

Reply:
Re-use of packet codes made it read a bit weird so made it more explicit for clarity

Reply:
^ @cammellos